Userland/Libraries/LibJS/Runtime/RegExpPrototype.cpp

```diff
@@ -332,6 +335,27 @@ static ThrowCompletionOr<Value> regexp_builtin_exec(VM& vm, RegExpObject& regexp
         }
     }
 
+    // https://github.com/tc39/proposal-regexp-legacy-features#regexpbuiltinexec--r-s-
```

Comment:
If a function is from the main spec but implements changes from a proposal, we usually note that by adding both links and signatures at the top:

```cpp
// 22.2.5.2.2 RegExpBuiltinExec ( R, S ), https://tc39.es/ecma262/#sec-regexpbuiltinexec
// 22.2.5.2.2 RegExpBuiltInExec ( R, S ), https://github.com/tc39/proposal-regexp-legacy-features#regexpbuiltinexec--r-s-
static ThrowCompletionOr<Value> regexp_builtin_exec(VM& vm, RegExpObject& regexp_object, Utf16String string)
```

Reply:
done